cloud_run_jobs_create/main.tf

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+provider "google-beta" {
+  region = "us-central1"
+}
+
+# Project data
+data "google_project" "project" {
+  project_id = "your-project-id"
```

Comment:
Please remove the `project_id` attribute.

This works just as well:

```
data "google_project" "project" {
}
```

Reply:
Maybe you can omit the `data` block completely, as it should only be needed as a workaround. Please see https://cloud.google.com/docs/terraform/basic-commands#google_project